dockers/docker-lldp/lldpmgrd

```diff
@@ -100,31 +109,24 @@ class LldpManager(daemon_base.DaemonBase):
     def is_port_up(self, port_name):
         """
         Determine if a port is up or down by looking into the oper-status for the port in
-        PORT TABLE in the Application DB
+        PORT TABLE in the State DB
         """
         # Retrieve all entires for this port from the Port table
-        (status, fvp) = self.app_port_table.get(port_name)
+        (status, fvp) = self.state_port_table.get(port_name)
         if status:
             # Convert list of tuples to a dictionary
             port_table_dict = dict(fvp)
 
             # Get the oper-status for the port
-            if "oper_status" in port_table_dict:
-                port_oper_status = port_table_dict.get("oper_status")
-                self.log_info("Port name {} oper status: {}".format(port_name, port_oper_status))
+            if "netdev_oper_status" in port_table_dict:
+                port_oper_status = port_table_dict.get("netdev_oper_status")
                 return port_oper_status == "up"
             else:
                 return False
         else:
-            # Retrieve PortInitDone entry from the Port table
```

Comment:
should we log info for debug purpose?

Reply:
in case the port is not up or not exist on table we have log info for it outside the function